lib/livebook_web/live/hub/edit_live.ex

```diff
@@ -14,7 +14,7 @@ defmodule LivebookWeb.Hub.EditLive do
 
   @impl true
   def handle_params(params, _url, socket) do
-    Hubs.subscribe([:connection, :secrets])
+    Hubs.subscribe([:connection, :secrets, :file_systems])
```

Comment:
This should be on mount. If we do it on `handle_params`, we end-up subscribing every time the URL changes.